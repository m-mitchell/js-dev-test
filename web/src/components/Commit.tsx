import React from 'react';

const Commit = ({commitDate, commitAuthor, commitMessage}) => (
  <tr>
    <td>{commitDate}</td>
    <td>{commitAuthor}</td>
    <td>{commitMessage}</td>
  </tr>
)

export default Commit;
