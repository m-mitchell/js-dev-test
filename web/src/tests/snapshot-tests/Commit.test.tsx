import React from 'react';
import Commit from '../../components/Commit';
import renderer from 'react-test-renderer';

test('Create Commits Table TR w/ (Date, Author, Message)', () => {
  const component = renderer.create(

    <Commit
      commitDate="2018-10-24T18:50:13Z"
      commitAuthor="Michael Gauthier"
      commitMessage="Merge pull request #1 from nburka/gauthierm-l10n\n\nbetter padding on language controls" />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

})
