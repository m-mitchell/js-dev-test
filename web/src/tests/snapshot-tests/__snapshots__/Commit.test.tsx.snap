// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Commits Table TR w/ (Date, Author, Message) 1`] = `
<tr>
  <td>
    2018-10-24T18:50:13Z
  </td>
  <td>
    Michael Gauthier
  </td>
  <td>
    Merge pull request #1 from nburka/gauthierm-l10n\\n\\nbetter padding on language controls
  </td>
</tr>
`;
